charts/prometheus-kafka-exporter/values.yaml

```
@@ -1,6 +1,6 @@
 image:
   repository: danielqsj/kafka-exporter
-  tag: v1.4.2
+  tag: ""
```

please add the info too...
```suggestion
  # Overrides the image tag whose default is the chart appVersion.
  tag: ""
```